src/index.js

```diff
@@ -44,7 +44,13 @@ export default function babel ( options ) {
 		},
 
 		load ( id ) {
-			if ( id === HELPERS ) {
+			if ( id !== HELPERS ) {
+				return;
+			}
+
+			const babelsMajor = parseInt(babelVersion[0], 10);
```

Comment:
You can just do `parseInt(babelVersion, 10)` since it'll stop parsing at the period. Plus then it won't break when we release Babel 10.x :P

Reply:
ah, yeah, good point, fixed